configure

```diff
@@ -826,6 +826,91 @@ EOF
   fi
 fi
 
+# test to see if Power8+ implementation is compile time possible
+echo >> configure.log
```

Comment:
Maybe you forgot about the CMake buildsystem?

I personally think that any new feature should be added only in the CMakeLists.txt (and consider the configure script some kind of minimal maintenance mode for legacy systems).

Reply:
I was unsure as to how up to date this was. I can add this configuration easy enough. Thanks for the advice.